kratos/python_scripts/apply_scalar_constraint_table_process.py

```diff
@@ -0,0 +1,32 @@
+import KratosMultiphysics
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyScalarConstraintTableProcess(Model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class ApplyScalarConstraintTableProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings ):
+        KratosMultiphysics.Process.__init__(self)
+
+        model_part = Model[settings["model_part_name"].GetString()]
+
+        params = KratosMultiphysics.Parameters("{}")
+        params.AddValue("model_part_name",settings["model_part_name"])
+        params.AddValue("variable_name",settings["variable_name"])
+        if settings.Has("is_fixed"):
+            params.AddValue("is_fixed",settings["is_fixed"])
+        
+        params.AddValue("value",settings["value"])
+        if settings["table"].GetInt() == 0:
+            self.process = KratosMultiphysics.ApplyConstantScalarValueProcess(model_part, params)
+        else:
+            params.AddValue("table",settings["table"])
+            self.process = KratosMultiphysics.ApplyDoubleTableProcess(model_part, params)
+
+    def ExecuteInitialize(self):
+        self.process.ExecuteInitialize()
+
+    def ExecuteInitializeSolutionStep(self):
+        self.process.ExecuteInitializeSolutionStep()
```

Comment:
you could return the process directly in the factory like I did [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/postprocess_eigenvalues_process.py) to reduce code-duplication